core/src/main/java/com/linecorp/armeria/server/AnnotatedHttpServices.java

```diff
@@ -69,6 +75,14 @@
     private static final Logger logger = LoggerFactory.getLogger(AnnotatedHttpServices.class);
 
     /**
+     * A {@link DecoratingServiceFunction} map for reusing.
+     */
+    private static final ConcurrentMap<
+            Class<? extends DecoratingServiceFunction<HttpRequest, HttpResponse>>,
+            DecoratingServiceFunction<HttpRequest, HttpResponse>>
+            DECORATING_SERVICE_FUNCTIONS = new ConcurrentHashMap<>();
```

Comment:
I guess we don't need use a uppercase name here because it's not a constant but mutable state.